src/pages/Follow/components/Follower.tsx

```diff
@@ -21,43 +27,91 @@ const useAcceptFollow = () => {
   const { mutate: accept } = useMutation('accept', (id: number) => acceptFollow({ id }), {
     onSuccess: () => {
       queryClient.invalidateQueries('received');
+      queryClient.invalidateQueries('follower');
     },
   });
-  return { accept };
+  return accept;
+};
+export const useDeleteFollow = () => {
+  const queryClient = useQueryClient();
+  const { mutate: del, data } = useMutation('delete', (account: string) => deleteFollow({ userAccount: account }), {
+    onSuccess: () => {
+      queryClient.invalidateQueries('follower');
+    },
+  });
+
+  return { del, data };
+};
+
+const useCancelFollow = () => {
+  const queryClient = useQueryClient();
+  const { mutate: cancel } = useMutation('cancel', (id: number) => cancelFollow({ id }), {
+    onSuccess: () => {
+      queryClient.invalidateQueries('sended');
+      queryClient.invalidateQueries('search');
+    },
+  });
+
+  return cancel;
 };
 
 interface Props {
   account: string,
   nickname: string,
   followedType: string,
   id: number,
+  requestId?: number,
 }
 
 export default function Follower({
-  nickname, account, followedType, id,
+  nickname, account, followedType, id, requestId,
 }: Props) {
   const request = useRequestAndUpdate();
-  const { accept } = useAcceptFollow();
-
+  const accept = useAcceptFollow();
+  const { del } = useDeleteFollow();
+  const cancel = useCancelFollow();
+  const navigate = useNavigate();
   return (
-    <div className={style.follower}>
+    <div className={style.follower} id={`${id}`}>
       <img className={style.follower__image} src={defaultImage} alt="default" />
       <div className={style.follower__content}>
-        <p className={cn({ [style['follower__content--font']]: true })}>{nickname}</p>
-        <p>{account}</p>
+        <button
+          type="button"
+          className={cn({ [style['follower__content--font']]: true })}
```

Comment:
font 대신 nickname 어떤가요?
```suggestion
          className={cn({ [style['follower__content--nickname']]: true })}
```

Reply:
좋습니다